modules/data_integrity_flag/php/data_integrity_flag.class.inc

```diff
@@ -73,13 +73,15 @@ class Data_Integrity_Flag extends \NDB_Menu_Filter
 
         if (!isset($req['instrument'])
             || !isset($req['visitLabel'])
-            || !isset($req['comment'])
             || !isset($req['date'])
             || !isset($req['flagStatus'])
         ) {
             return "Error";
         }
         $req = \Utility::asArray($req);
+        // comment may not be set since it is not a required field
+        $comment = (isset($req['comment']) && !empty($req['comment'])) ?
```

Comment:
```suggestion
        $comment = !empty($req['comment']) ?
```

@driusan just tuaght me that `empty()` will actually check for a variable being set all on its own, so I don't think you need it here.

Reply:
@jesscall ping